devlink_test.go

```diff
@@ -0,0 +1,34 @@
+// +build linux
+
+package netlink
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+func setupDevlinkKModule(t *testing.T, name string) {
```

Comment:
use this method: https://github.com/vishvananda/netlink/blob/master/netlink_test.go#L118

Reply:
@fcrisciani done